Add TimetableLunchItemDecoration

itome opened the pull request:
## Issue
- close #620 

## Overview (Required)
- Implement TimetableLunchItemDecoration to overwrite luch row.
- Find lunch item from adapter and draw on the view in `onDrawOver`.

## Links
-

## Screenshot
Before | After
:--: | :--:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/24409457/52142152-3dcad480-269b-11e9-8778-47f679db049c.gif) | ![ezgif com-video-to-gif 1](https://user-images.githubusercontent.com/24409457/52142182-52a76800-269b-11e9-9fd2-76d5aa2467c2.gif)

takahirom commented:
@MoyuruAizawa If you have any opinion for this PR. Please comment it 🙏

takahirom commented on feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/TimetableLunchDecoration.kt:
```diff
@@ -0,0 +1,127 @@
+import java.util.concurrent.TimeUnit
+
+class TimetableLunchDecoration(
+    private val timeLabelWidth: Float,
```
Why do you use parameter for it? ⛱ 
I think you can just use field like this.

```
class TimetableLunchDecoration(context: Context, groupAdapter: GroupAdapter<*>) {
    private val timeLabelWidth: Float = context.resources.getDimension(R.dimen.tabular_form_time_label_width)
```

takahirom commented:
Can you fix this? 👀 
![lunch](https://user-images.githubusercontent.com/1386930/52173858-77046100-27cf-11e9-93a5-45a13827a7cf.gif)

itome commented:
I fixed two bugs
- Text appears out of lunch item
- Lunch decoration does'nt appear in ROOM5, ROOM6, in Day1

And I did little hacky for fixing it.
I added lunch items for all rooms before adding groupie Items.

takahirom commented:
Can you fix the conflict? 👀

takahirom commented:
Thanks 👍
